pkg/controller/core/localqueue_controller.go

```diff
@@ -274,41 +281,47 @@ func (r *LocalQueueReconciler) initializeAdmissionFsStatus(lq *kueue.LocalQueue)
 }
 
 func (r *LocalQueueReconciler) reconcileConsumedUsage(ctx context.Context, lq *kueue.LocalQueue) error {
+	lqKey := utilqueue.Key(lq)
 	halfLifeTime := r.admissionFSConfig.UsageHalfLifeTime.Seconds()
-	// reset usage to 0 if halfLife is 0
+	now := r.clock.Now()
+
 	if halfLifeTime == 0 {
-		return r.updateAdmissionFsStatus(ctx, lq, corev1.ResourceList{})
+		r.log.V(2).Info("Reset AFS consumed resources cache", "namespace", lq.Namespace, "name", lq.Name)
```

Comment:
This log should be after the request (or even maybe cache to be more future-proof) success